src/lib/schematics/BUILD.bazel

```diff
@@ -1,21 +1,56 @@
 package(default_visibility = ["//visibility:public"])
 
 load("@build_bazel_rules_typescript//:defs.bzl", "ts_library")
-load("@build_bazel_rules_nodejs//:defs.bzl", "npm_package")
+load("@build_bazel_rules_nodejs//:defs.bzl", "npm_package", "jasmine_node_test")
 
+filegroup(
+  name = "schematics_assets",
+  srcs = glob(["**/files/**/*"]) + ["README.md", "collection.json", "migration.json"],
+)
 
 ts_library(
   name = "schematics",
-  module_name = "@angular/material/schematics",
```

Comment:
I remember needing to add this for something... can you confirm that `//src/lib:npm_package` still outputs the right thing without this?

Reply:
Can confirm that this seems to be necessary in order to have more reasonable `AMD` module names .

Also found an issue with the TS rules within Windows: https://github.com/bazelbuild/rules_typescript/pull/239